build.ps1

```diff
@@ -302,14 +304,15 @@ try {
         $oldVerbosity = $verbosity
         $verbosity = "normal"
 
-        if (IsWindowsDevelopmentBox)
+        if (IsWindowsDesktopDevelopmentBox)
         {
             RunTests e2e\test "End-to-end tests (NetCoreApp2.1, NET47, NET451)" "*" "TestCategory!=IoTHub-FaultInjection-PoolAmqp"
             RunTests e2e\test "End-to-end tests (NetCoreApp2.1, NET47, NET451)" "*" "TestCategory=IoTHub-FaultInjection-PoolAmqp"
         }
         else
         {
-            RunTests e2e\test "End-to-end tests (NetCoreApp2.1)" "netcoreapp2.1"
+            # RunTests e2e\test "End-to-end tests (NetCoreApp2.1)" "netcoreapp2.1"
+            RunTests e2e\test "End-to-end tests (NetCoreApp2.1)" "netcoreapp2.1" "FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_InvalidIdScope_Register_AmqpWs_Tpm_Fail&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_InvalidIdScope_Register_Amqp_Tpm_Fail&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_InvalidIdScope_Register_Http_Tpm_Fail&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_InvalidRegistrationId_TpmRegister_Amqp_Fail&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_InvalidRegistrationId_TpmRegister_Http_Fail&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_ValidRegistrationId_AmqpWs_Tpm_RegisterOk&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_ValidRegistrationId_Amqp_Tpm_RegisterOk&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_ValidRegistrationId_HttpWithNullProxy_Tpm_RegisterOk_Individual&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_ValidRegistrationId_HttpWithProxy_Tpm_RegisterOk_IndividualEnrollment&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningE2ETests.ProvisioningDeviceClient_ValidRegistrationId_Http_Tpm_RegisterOk&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningServiceClientE2ETests.ProvisioningServiceClient_Tpm_IndividualEnrollments_Create_HttpWithProxy_Ok&FullyQualifiedName!=Microsoft.Azure.Devices.E2ETests.ProvisioningServiceClientE2ETests.ProvisioningServiceClient_Tpm_IndividualEnrollments_Create_HttpWithoutProxy_Ok"
```

Comment:
This comment needs to be removed. Other than that, this PR looks good.

Reply:
Done.